com.unity.robotics.urdf-importer/Runtime/RosSharpDefinitions/Sensor.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Xml;
+using System.Xml.Linq;
+using System;
+using System.Numerics;
+
+
+namespace RosSharp.Urdf
+{
+    public class Sensor
+    {
+        public string name
+        { get; set; }
+        public string type
+        { get; set; }
+        public string topic
+        { get; set; }
+        private static string k_baseKey = "sensor";
+        private static string k_attributeDelimit = "@";
+        private static string k_elementDelimit = "/";
```

Comment:
naming convention from [these code standards](https://ono.unity3d.com/unity-extra/unity-meta/raw/@/ReferenceSource/CSharp/Assets/CSharpReference.cs) is prefix_PascalCase
```
        private static string k_BaseKey = "sensor";
        private static string k_AttributeDelimit = "@";
        private static string k_ElementDelimit = "/";
```